chain/consensus/tspow/consensus.go

```diff
@@ -0,0 +1,34 @@
+package tspow
+
+import (
+	"context"
+	"github.com/filecoin-project/go-state-types/abi"
```

Comment:
The packages are not sorted. We use the following policy: native, external, filecoin-related

Reply:
Done